### ISSUE TITLE
Can't load custom textures

### ISSUE THREAD

**Issue by themagpieswarble** [Rosalie241/RMG#75]
I'm attempting to load Mario Kart 64 HD custom textures with my Mario Kart 64 Rom.  I have the Mario Kart 64 HD texture pack in this folder 'C:\Users\User\Documents\Rosalie Mupen\Data\hires_texture\MARIOKART64' and in my Rosalie GlideN64 settings I have 'Use texture pack' enabled under the 'Texture Enhancement' tab with the Texture pack path set to: 'C:/Users/User/Documents/Rosalie Mupen/Data/hires_texture'

When I start up the game, it doesn't scan and start installing/loading the textures. It just quickly loads into Mario Kart 64 as normal.

Am I missing something obvious? Thanks!

**Comment by Rosalie241** [comment]
Sadly it seems to work fine for me, so I'm not sure what the issue is but here are a few things you can check/try:

Do you have `use file storage instead of memory cache` enabled? (cos you should)
Are you certain you extracted the texture pack correctly? i.e does it look like inside the `MARIOKART64` directory there are images and subdirectories?

**Comment by themagpieswarble** [comment]
Yes, the unzipped file looks good. Full of separate sub-folders and images in each. I tried making a sub-folder called RMG like you have to see if that mattered but didn't seem to help. Also tried checking that box 'Use file storage'... and no dice. 

No idea what I'm doing wrong. Here's some screenshots:

https://imgur.com/a/lSCo4T0

Anything look wrong to you? Thanks!

**Comment by GhostlyDark** [comment]
If the path is correct, try moving RMG and the texture pack files into an unprotected directory, away from any folders that are created and used by Windows. Be mindful that per-game settings are enabled by default - so the settings before you start Mario Kart 64 are different from the settings after booting the game up.

If the issue persists, please post your full GLideN64 settings here.

**Comment by themagpieswarble** [comment]
**"Be mindful that per-game settings are enabled by default - so the settings before you start Mario Kart 64 are different from the settings after booting the game up."**

That was it! Per-Game Settings was enabled and when I ran Mario Kart 64 it loaded the previous default settings which didn't include enabling custom textures. After booting Mario Kart 64, I went back into Graphic Settings and turned on all the texture settings and it immediately started loading the custom textures for Mario Kart 64 HD. Thank you so much!
